### PR TITLE
Add endpoint for populating users and movies

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: horse/web/populate.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from horse import models
+from .schemas.populate import populate_schema
+
+populate_bp = Blueprint('populate_api', __name__)
+populate_api = Api(populate_bp)
+
+
+class Populate(Resource):
+    def post(self):
+        self._clear_repositories()
```
Comment (biern) [inline horse/web/populate.py]: I think clearing repositories should happen only once the schema is validated

File: tests/integration/test_populate.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    assert 'Andrew' in user_names
+    assert 'John' in user_names
+
+    user = users_repo.get_by_name('John')
+    followed_names = [followed.name for followed in user.get_followed_users()]
+    assert 'Andrew' in followed_names
+
+    liked_titles = [liked.title for liked in user.get_liked_movies()]
+    assert 'Home alone' in liked_titles
+    assert 'Shining' in liked_titles
```
Comment (biern) [inline tests/integration/test_populate.py:77]: This test is pretty large and could be easily split into several smaller scenarios. 
Though I'm not confident we should put more effort here, it's just a dirty, unsecured test-only endpoint anyway

File: tox.ini [hunk 3]
```diff
@@ -2,8 +2,9 @@
 envlist=py34
 
 [testenv]
-commands=python -m pytest --flakes --pep8 {posargs}
+commands=python -m pytest --flakes --pep8 --cov=horse {posargs}
```
Comment (biern) [inline tox.ini:5]: 👍

**Review by biern — CHANGES_REQUESTED** [review]

File: tests/integration/test_populate.py (hunk 2 above)
Comment (pgrzesik) [reply tests/integration/test_populate.py:77]: I also don't like the fact that this test is pretty big, I will split it into few smaller tests

File: horse/web/populate.py (hunk 1 above)
Comment (pgrzesik) [reply horse/web/populate.py]: agreed

**Pull request merged**